src/lib/storage/table.hpp

```diff
@@ -72,6 +73,13 @@ class Table : private Noncopyable {
   void append(const std::vector<AllTypeVariant>& values);
 
  protected:
-  // Implementation goes here
+  ChunkOffset _chunk_size;
```

Comment:
Naming could cause confusion as this variable is holding the max. chunk size which could be confused with the current chunk size.